### DISCUSSION TITLE
How to convert identical coordinate system to custom coordinate system

### DISCUSSION THREAD

**Discussion by TheVisionz** [pyproj4/pyproj#911] [Q&A]
Hello, currently I am using Pyproj to convert coordinate from EPSG:4326 to others such as EPSG:6675 and I can get the coordinate correctly from the conversion. But right now, I am trying to use Local coordinate ( own coordinate system ) which I want to convert from EPSG:6675 to it.
I go online and search for transformation and I saw a lot formula but suddenly I think Pyproj able to convert from one EPSG to others, so It might has the basic formula to convert those coordinate system. 
Is it possible for me to know about the conversion formula to transform from one EPSG coordinate to others ?

Thank you

**Comment by snowman2** [comment]
Can you define what you mean by "formula"?
